sdk/src/Services/DynamoDBv2/Properties/AssemblyInfo.cs

```diff
@@ -20,6 +19,8 @@
 #else
 #error Unknown platform constant - unable to set correct AssemblyDescription
 #endif
+[assembly: InternalsVisibleTo("AWSSDK.UnitTests.DynamoDBv2.Net35, PublicKey=0024000004800000940000000602000000240000525341310004000001000100db5f59f098d27276c7833875a6263a3cc74ab17ba9a9df0b52aedbe7252745db7274d5271fd79c1f08f668ecfa8eaab5626fa76adc811d3c8fc55859b0d09d3bc0a84eecd0ba891f2b8a2fc55141cdcc37c2053d53491e650a479967c3622762977900eddbf1252ed08a2413f00a28f3a0752a81203f03ccb7f684db373518b4")]
```

Comment:
The `AssemblyInfo.cs` file is generated so this change would get whipped out the next time we do a build. You would need to add that change to the `tt` file in the generator here and then parameterize the service name. https://github.com/aws/aws-sdk-net/blob/02ec2a6188696a765eebbf50c5fd0bf2f1321b11/generator/ServiceClientGeneratorLib/Generators/SourceFiles/AssemblyInfo.tt

To run the generator you open the [AWSSDKGenerator.sln](https://github.com/aws/aws-sdk-net/blob/02ec2a6188696a765eebbf50c5fd0bf2f1321b11/generator/AWSSDKGenerator.sln) solution and then run the `ServiceClientGenerator` project. That will regenerate the the `AssemblyInfo.cs`. You don't want to include the generated code in the PR because then the PR will be huge.

Reply:
Is it possible to add those attributes conditionally in that tt template? Don't want to add those attributes to every single project...

I'm thinking if it would make sense simply to create a separate file InternalVisibleToAttributes.cs instead.

Reply:
@normj 
I have updated the generator to sign UnitTests projects, and include them in InternalsVisibleTo attributes.
Let me know if you'd prefer any other approach.

Thanks for the help!

Reply:
@normj 
The other option would be to remove this IVT attribute, and add unit tests for DynamoDBEntryConversion itself.
E.g. test that DynamoDBEntryConversion is able to convert to and from `byte` instead of testing that ByteConverter is registered.
(I wasn't able to find existing tests for that, correct me if I'm wrong).

On its own, that wouldn't prevent from adding a converter and forgetting to add it to the schema, but hopefully it will encourage to add corresponding tests.

WDYT?